presto-main/src/main/java/com/facebook/presto/sql/analyzer/QueryExplainer.java

```diff
@@ -188,7 +194,7 @@ public Plan getLogicalPlan(Session session, Statement statement, List<Expression
         // analyze statement
         Analysis analysis = analyze(session, statement, parameters, warningCollector);
         // plan statement
-        LogicalPlanner logicalPlanner = new LogicalPlanner(true, session, planOptimizers, idAllocator, metadata, sqlParser, statsCalculator, costCalculator, warningCollector);
+        LogicalPlanner logicalPlanner = new LogicalPlanner(true, session, planOptimizers, idAllocator, metadata, sqlParser, statsCalculator, costCalculator, warningCollector, planChecker);
```

Comment:
nit: This line's getting long. it's probably about time to split this into one line per-argument.